Directory.Build.targets

```diff
@@ -25,11 +25,19 @@
   <!-- Packaging -->
   <PropertyGroup>
     <PackageDescription Condition="'$(PackageDescription)' == '' and '$(Description)' != ''">$(Description)</PackageDescription>
-    <PackageDescription Condition="'@(PackageDescriptionCommonType)' != '' and '$(PackageDescription)' != ''">$(PackageDescription)%0A%0ACommonly Used Types:@(PackageDescriptionCommonType->'%0A%(Identity)')</PackageDescription>
     <RuntimePackageDisclaimer>Internal implementation package not meant for direct consumption. Please do not reference directly.</RuntimePackageDisclaimer>
     <UseRuntimePackageDisclaimer Condition="'$(IsInternalRuntimePackage)' == '' and
                                             ($(MSBuildProjectName.StartsWith('runtime.native')) or '$(PackageTargetRuntime)' != '')">true</UseRuntimePackageDisclaimer>
     <PackageDescription Condition="'$(PackageDescription)' == '' and '$(UseRuntimePackageDisclaimer)' == 'true'">$(RuntimePackageDisclaimer)</PackageDescription>
     <PackageDescription Condition="'$(PackageDescription)' != '' and '$(UseRuntimePackageDisclaimer)' == 'true'">$(RuntimePackageDisclaimer) %0A$(PackageDescription)</PackageDescription>
   </PropertyGroup>
+
+  <Target Name="AppendDescriptionCommonTypesToPackageDescription"
```

Comment:
would it make sense to move this target to the packaging sdk in arcade repo ?

Reply:
I don't think so. This is just a mutation of the PackageDescription property and you usually want to have that near the code that set the PackageDescription for more flexibility.